packages/reports/addon/components/filter-builders/collapsed.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{filter-builders/collapsed
+ *       displayName=displayName
+ *       filter=filter
+ *       request=request
+ *       field=field
+ *   }}
+ */
+import Component from '@ember/component';
+import layout from 'navi-reports/templates/components/filter-builders/collapsed';
+
+export default Component.extend({
```

Comment:
Could be a native class
https://github.com/yahoo/navi/blob/289a5225f311da729865a689618852fa17acc1d6/packages/reports/addon/components/navi-request-column-config/base.js#L15-L23